frontend/pages/queries/ManageQueriesPage/_styles.scss

```diff
@@ -184,5 +184,10 @@
         }
       }
     }
+
+    .query-icon {
```

Comment:
It seems like this pushes the icon a bit too far down, do you agree @lukeheath?
![Screenshot 2023-05-23 at 2 06 33 PM](https://github.com/fleetdm/fleet/assets/61553566/de7535d6-760e-4064-b408-bbde386613af)

Reply:
It looks like this for me in Chrome and Safari: 
![image](https://github.com/fleetdm/fleet/assets/2495927/6a817236-0920-4b90-9a8f-c5bb45fb2e92)

Reply:
It actually seems to be a zoom thing on Firefox – lgtm in all 3 browsers at 100%, then off as you zoom in/out in FF (80, 90, 110%)